superset-frontend/src/chart/Chart.jsx

```diff
@@ -19,12 +19,12 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import Alert from 'src/components/Alert';
-import { styled, logging } from '@superset-ui/core';
+import { styled, logging, t } from '@superset-ui/core';
 
 import { isFeatureEnabled, FeatureFlag } from 'src/featureFlags';
-import { Logger, LOG_ACTIONS_RENDER_CHART } from '../logger/LogUtils';
+import Button from 'src/components/Button';
 import Loading from '../components/Loading';
```

Comment:
```suggestion
import Loading from 'src/components/Loading';
```
might as well add `src` alias paths for the rest of these imports while we're here (I won't point out the rest... you get the idea) :)